File: NModM/py3/nmodm.py
# N Mod M solution in Python 3 for CodeEval.com by Steven A. Dunn

import sys

for line in open(sys.argv[1], 'r'):
  line = line.rstrip('\n').split(',')
  N = int(line[0])
  M = int(line[1])

  x = int(N / M)
  y = x * M
  mod = N - y

  print(mod)

File: NModM/cpp/NModM.cpp
// N Mod M solution in C++ for CodeEval.com by Steven A. Dunn

#include <cstdlib>
#include <fstream>
#include <iostream>
#include <string>

using namespace std;

int main(int argc, char* argv[])
{
	ifstream inputFile(argv[1]);
	string line;
	
	if (inputFile)
	{
		while (getline(inputFile, line))
		{
			size_t commaPos = line.find(",");
			int n = atoi(line.substr(0, commaPos).c_str());
			int m = atoi(line.substr(commaPos + 1).c_str());

			int x = n / m;
			int y = x * m;
			int mod = n - y;
			
			cout << mod << endl;
		}
	}
}